app/views/register/admin/ircc-counts.js

```diff
@@ -61,13 +61,13 @@ define(['app', 'lodash', 'js/common', 'moment', 'components/scbd-angularjs-contr
                     }
 
                     if($scope.filters.startDate || $scope.filters.endDate) {
-                        var startDate = $scope.filters.startDate ? $scope.filters.startDate + 'T00:00:00.000Z' : '*';
-                        var endDate = $scope.filters.endDate ? $scope.filters.endDate + 'T23:59:59.999Z' : '*';
+                        var startDate   = solr.escape($scope.filters.startDate ? $scope.filters.startDate + 'T00:00:00.000Z' : '*');
+                        var endDate     = solr.escape($scope.filters.endDate ? $scope.filters.endDate + 'T23:59:59.999Z' : '*');
```

Comment:
is the star `*` must be escaped?

Reply:
no because it has a meaning in solr, if escaped it will be treated as string query

Reply:
then you should change your code because `'*'` is inside the parenthesis of the escape function 
```suggestion
                        var startDate   = $scope.filters.startDate ? solr.escape($scope.filters.startDate + 'T00:00:00.000Z') : '*';
                        var endDate     = $scope.filters.endDate ?  solr.escape($scope.filters.endDate + 'T23:59:59.999Z') : '*';
```